src/Routes/Home/Home.jsx

```diff
@@ -0,0 +1,14 @@
+import './Home.css'
+import creepyClerk from '../../assets/Igor_Persona5.gif'
+
+export default function Home(){
```

Comment:
Prefiero que se acostumbren a usar named exports en lugar de default exports. Asi se aseguran que se use importe con el mismo nombre en todo lados
```suggestion
export function Home(){
```